docs/upgrading-eck.asciidoc

```diff
@@ -7,131 +7,23 @@ endif::[]
-   updateStrategy:
-     changeBudget:
-       maxUnavailable: 1
--    groups: []
-   secureSettings:
--    secretName: "gcs-credentials"
-+  - secretName: "gcs-credentials"
--  nodes:
--  - nodeCount: 3
-+  nodeSets:
-+  - count: 3
-+    name: master-nodes
-----
+[id="{p}-ga-openshift"]
+=== Upgrades in earlier Kubernetes and OpenShift versions
 
+Upgrading from ECK 1.0.0-beta1 to 1.0.0 has some special considerations for users running either:
+- Kubernetes versions < 1.13
+- OpenShift 3.11
 
-.Kibana
-* Replace `v1alpha` in the `apiVersion` field with `v1beta1`
-* Rename `nodeCount` to `count`
-* If you specified `secureSettings`, convert `secretName` into an array item by prefixing it with `-`
-
-[source,patch,subs="attributes"]
-----
--apiVersion: kibana.k8s.elastic.co/v1alpha1
-+apiVersion: kibana.k8s.elastic.co/v1beta1
- kind: Kibana
- metadata:
-   name: kibana-sample
- spec:
-   version: {version}
--  nodeCount: 1
-+  count: 1
-   elasticsearchRef:
-     name: "elasticsearch-sample"
-   secureSettings:
--    secretName: "gcs-credentials"
-+  - secretName: "gcs-credentials"
-----
-
-
-.APM Server
-* Replace `v1alpha` in the `apiVersion` field with `v1beta1`
-* Rename `nodeCount` to `count`
-* If you specified `secureSettings`, convert `secretName` into an array item by prefixing it with `-`
-
-[source,patch,subs="attributes"]
-----
--apiVersion: apm.k8s.elastic.co/v1alpha1
-+apiVersion: apm.k8s.elastic.co/v1beta1
- kind: ApmServer
- metadata:
-   name: apm-server-sample
- spec:
-   version: {version}
--  nodeCount: 1
-+  count: 1
-   elasticsearchRef:
-     name: "elasticsearch-sample"
-   secureSettings:
--    secretName: "gcs-credentials"
-+  - secretName: "gcs-credentials"
-----
-
-[float]
-[id="{p}-1.0.0-beta1-backwards-compatibility"]
-==== Backwards compatibility
-
-The 1.0.0-beta version of the operator does not delete resources created by older versions of the operator, but it also does not manage them. Attempting to delete resources created with a 0.9.0 version will hang if ECK 1.0.0-beta1 is running. To unblock the deletion, remove any registered finalizer from the resource (substituting the correct name for `quickstart`):
+In this case, ECK must be uninstalled and reinstalled (due to link:https://github.com/kubernetes/kubernetes/issues/73752[an upstream bug] that was not backported). To uninstall ECK, see <<{p}-uninstalling-eck>>. Then you can follow the <<{p}-quickstart>> to install anew. You may wish to save your existing `v1beta1` manifests and update them to the `v1` API version, and to <<{p}-snapshots,take snapshots>> and restore them as well.
 
-[source,sh]
-----
-kubectl patch elasticsearch quickstart --patch '{"metadata": {"finalizers": []}}' --type=merge
-kubectl patch kibana quickstart --patch '{"metadata": {"finalizers": []}}' --type=merge
-kubectl patch apmserver quickstart --patch '{"metadata": {"finalizers": []}}' --type=merge
-----
+Once upgraded, you should use the `v1` API version for all Elastic manifests.
```

Comment:
Agreed with @alaudazzi , this is a requirement, not sure that `should` is appropriate here

Reply:
Ah good point, it's just "should" for >=1.13